src/semgrep_agent/semgrep.py

```diff
@@ -187,9 +187,21 @@ def get_findings(
                 for e in findings.errors:
                     for s in render_error(e):
                         click.echo(f"|    {s}", err=True)
+            inventory_findings_len = 0
+            for finding in findings.current:
```

Comment:
Might be useful to factor into a function to count cai findings given list of findings